client/app/components/packages/landuse-form/lead-agency.js

```diff
@@ -18,6 +19,13 @@ export default class PackagesLanduseFormLeadAgencyComponent extends Component {
 
   get chosenAccountId() {
     const account = this.args.accounts.find((account) => account.name === this.chosenAccountName);
-    return account.id;
+    return account ? account.id : null;
+  }
+
+  @action
+  clearDropdown(data) {
+    this.chosenAccountName = null;
+    data.set('chosenLeadAgencyId', null);
+    data.set('leadAgency', null);
```

Comment:
.set() and .get() are changeset methods for modifying the changeset

Reply:
I know, but changesets are object proxies, so native setting/getting should work, no?

```
data.chosenLeadAgencyId = null;
data.leadAgency = null;
```

Does this native syntax not work?

Reply:
Ah, I think I was just wanting to be safe and go with whats in the API. The native setting/getting seems to work but I wasn't sure since it wasn't on the docs. I'll change them for consistency since they seem okay and we have a second opinion